timvt/endpoints/tiles.py

```diff
@@ -70,7 +78,15 @@ async def tile(
         async with db_pool.acquire() as conn:
             q = await conn.prepare(sql_query)
             content = await q.fetchval(
-                bbox.xmin, bbox.ymin, bbox.xmax, bbox.ymax, epsg, segSize
+                bbox.xmin,  # 1
+                bbox.ymin,  # 2
+                bbox.xmax,  # 3
+                bbox.ymax,  # 4
+                epsg,  # 5
+                segSize,  # 6
+                TILE_RESOLUTION,  # 7
+                TILE_BUFFER,  # 8
+                MAX_FEATURES_PER_TILE,  # 9
```

Comment:
I think you meant to use `limit` here not `MAX_FEATURES_PER_TILE` ?

Reply:
The limit string that is getting f string formatted is "LIMIT $9", so the $9 still needs to refer to MAX_FEATURES_PER_TILE for the sql engine parameter substitution.